### PR TITLE
feat: release 2.14.0

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by tzssangglass — APPROVED** [review]

**Review by tokers — DISMISSED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Review by tzssangglass — DISMISSED** [review]

File: docs/zh/latest/CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +57,62 @@ title: CHANGELOG
...
+    - [#7028](https://github.com/apache/apisix/pull/7028)
+    - [#7032](https://github.com/apache/apisix/pull/7032)
+- 引入一个名为 xRPC 的实验性框架来管理非 HTTP 的 L7 流量。
+    - [#6885](https://github.com/apache/apisix/pull/6885)
+    - [#6901](https://github.com/apache/apisix/pull/6901)
+    - [#6919](https://github.com/apache/apisix/pull/6919)
+    - [#6960](https://github.com/apache/apisix/pull/6960)
+    - [#6965](https://github.com/apache/apisix/pull/6965)
+    - [#7040](https://github.com/apache/apisix/pull/7040)
+- 现在我们支持在代理 Redistraffic 过程中根据命令和键添加延迟，它建立在 xRPC 之上。
```
Comment (leslie-tsang) [inline docs/zh/latest/CHANGELOG.md:78]:
```suggestion
- 现在我们支持在代理 Redis traffic 过程中根据命令和键添加延迟，它建立在 xRPC 之上。
```
Would be better ?

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -56,6 +57,62 @@ title: CHANGELOG
...
+    - [#6999](https://github.com/apache/apisix/pull/6999)
+- 引入实验性支持，通过 xDS 配置 APISIX。
+    - [#6614](https://github.com/apache/apisix/pull/6614)
+    - [#6759](https://github.com/apache/apisix/pull/6759)
+- 增加一个选项，像 servlet 一样规范化 URI。[#6984](https://github.com/apache/apisix/pull/6984)
```
Comment (leslie-tsang) [inline docs/zh/latest/CHANGELOG.md:83]:
```suggestion
- 增加 `normalize_uri_like_servlet` 配置选项，像 servlet 一样规范化 URI。[#6984](https://github.com/apache/apisix/pull/6984)
```
Would be better ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -56,6 +57,62 @@ title: Changelog
...
+    - [#6919](https://github.com/apache/apisix/pull/6919)
+    - [#6960](https://github.com/apache/apisix/pull/6960)
+    - [#6965](https://github.com/apache/apisix/pull/6965)
+    - [#7040](https://github.com/apache/apisix/pull/7040)
+- Now we support adding delay according to the command & key during proxying Redistraffic, which is built above xRPC:
+    - [#6999](https://github.com/apache/apisix/pull/6999)
+- Introduce an experimental support to configure APISIX via xDS:
+    - [#6614](https://github.com/apache/apisix/pull/6614)
+    - [#6759](https://github.com/apache/apisix/pull/6759)
+- Add an option to normalize uri like servlet: [#6984](https://github.com/apache/apisix/pull/6984)
```
Comment (leslie-tsang) [inline CHANGELOG.md:83]:
```suggestion
- Add `normalize_uri_like_servlet` option to normalize uri like servlet: [#6984](https://github.com/apache/apisix/pull/6984)
```
Ditto

File: CHANGELOG.md [hunk 4]
```diff
@@ -56,6 +57,62 @@ title: Changelog
...
+    - [#7028](https://github.com/apache/apisix/pull/7028)
+    - [#7032](https://github.com/apache/apisix/pull/7032)
+- Introduce an experimental framework called xRPC to manage non-HTTP L7 traffic:
+    - [#6885](https://github.com/apache/apisix/pull/6885)
+    - [#6901](https://github.com/apache/apisix/pull/6901)
+    - [#6919](https://github.com/apache/apisix/pull/6919)
+    - [#6960](https://github.com/apache/apisix/pull/6960)
+    - [#6965](https://github.com/apache/apisix/pull/6965)
+    - [#7040](https://github.com/apache/apisix/pull/7040)
+- Now we support adding delay according to the command & key during proxying Redistraffic, which is built above xRPC:
```
Comment (leslie-tsang) [inline CHANGELOG.md:78]:
```suggestion
- Now we support adding delay according to the command & key during proxying Redis traffic, which is built above xRPC:
```
Would be better ?

File: docs/zh/latest/CHANGELOG.md [hunk 5]
```diff
@@ -56,6 +57,62 @@ title: CHANGELOG
...
+- 通过 apisix-seed 实现 Zookeeper 服务发现：[#6751](https://github.com/apache/apisix/pull/6751)
+
+### Plugin
+
+- real-ip 插件支持像`real_ip_recursive`那样的递归 IP 搜索。[#6988](https://github.com/apache/apisix/pull/6988)
```
Comment (leslie-tsang) [inline docs/zh/latest/CHANGELOG.md:88]:
```suggestion
- real-ip 插件支持像 `real_ip_recursive` 那样的递归 IP 搜索。[#6988](https://github.com/apache/apisix/pull/6988)
```

File: docs/zh/latest/CHANGELOG.md [hunk 6]
```diff
@@ -56,6 +57,62 @@ title: CHANGELOG
...
+    - [#6859](https://github.com/apache/apisix/pull/6859)
+    - [#6854](https://github.com/apache/apisix/pull/6854)
+    - [#6853](https://github.com/apache/apisix/pull/6853)
+    - [#6846](https://github.com/apache/apisix/pull/6846)
+- DNS 支持端口为 0 的 SRV 记录：[#6739](https://github.com/apache/apisix/pull/6739)
+- 修复客户端 mTLS 在 TLS 会话重用中有时不生效的问题：[#6906](https://github.com/apache/apisix/pull/6906)
+- grpc-web 插件不会在响应中覆盖 Access-Control-Allow-Origin 头。[#6842](https://github.com/apache/apisix/pull/6842)
+- syslog 插件的默认超时已被纠正。[#6807](https://github.com/apache/apisix/pull/6807)
+- 修复 authz-keycloak 插件的 `access_denied_redirect_uri` 的设置有时不生效的问题。[#6794](https://github.com/apache/apisix/pull/6794)
+- 正确处理`USR2`信号。[#6758](https://github.com/apache/apisix/pull/6758)
```
Comment (leslie-tsang) [inline docs/zh/latest/CHANGELOG.md:110]:
```suggestion
- 正确处理 `USR2` 信号。[#6758](https://github.com/apache/apisix/pull/6758)
```

**Review by leslie-tsang — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]
LGTM, and please update PR's checklist, thanks!

**Comment by spacewander** [comment]
> LGTM, and please update PR's checklist, thanks!

Updated.

**Pull request merged**